packages/beeq/src/components/page-title/scss/bq-page-title.variables.scss

```diff
@@ -0,0 +1,10 @@
+/* -------------------------------------------------------------------------- */
+/*                   Page title custom properties                  */
+/* -------------------------------------------------------------------------- */
+
+:host {
+  /**
+   * @prop --bq-page-title--margin - Page title margin
+   */
+  --bq-page-title--margin: theme(spacing.m);
```

Comment:
Instead of adding a CSS custom property for margin, I would recommend adding:
```css
--text-color
--text-lineHeight
--text-size
--text-weight
```
which are more relevant to the scope of the component.